tests/unit/test_opentelemetry_tracing.py

```diff
@@ -0,0 +1,104 @@
+import sys
+import pytest
+
+from importlib import reload
+import mock
+
+from google.cloud.bigquery import opentelemetry_tracing, client
+
+from opentelemetry import trace
```

Comment:
I recommend using `pytest.importorskip` https://docs.pytest.org/en/latest/skipping.html#skipping-on-a-missing-import-dependency

This will allow the tests to run when opentelemetry is not installed.

Reply:
Then again, we probably need a test to check that things succeed when the library is not installed. See the https://github.com/googleapis/python-bigquery/blob/master/tests/unit/test__pandas_helpers.py tests for how to mock out imports to have a couple tests for the "no opentelemetry" path. See: https://github.com/googleapis/python-bigquery/blob/master/tests/unit/test__pandas_helpers.py#L954